mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Customizable unsubscribe link on mass mailing emails
```

Comment:
Put here also the same `====` to get a title with a higher scale (see https://github.com/OCA/maintainer-tools/pull/109)